tests/unit/test_routes_utils.py

```
@@ -507,3 +532,55 @@ def test_extract_uploaded_file_rejects_non_upload_values():
+    assert params["clean_audio"] is True
+
+
+def test_clean_audio_is_none_when_no_param_provided():
+    """params["clean_audio"] is None when none of the three params are present."""
+    params = _call_apply_prompt_flags({}, {})
+    assert params["clean_audio"] is None
+
+
+def test_extract_ext_extensionless_original_with_local_path():
+    """_extract_ext uses local_path extension when original_filename is extensionless."""
+    extract_ext = getattr(routes_utils, "_extract_ext")
```

Ruff (0.16.1)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)